guides/common/assembly_installing-capsule-server.adoc

```diff
@@ -19,7 +19,11 @@ endif::[]
 
 // Configuring Repositories
 ifdef::satellite[]
+:parent-dnf-module: {dnf-module}
+:dnf-module: satellite-capsule:el8
 include::modules/proc_configuring-repositories-proxy.adoc[leveloffset=+1]
+:dnf-module: {parent-dnf-module}
+:!parent-dnf-module:
```

Comment:
```suggestion
:parent-dnf-module!:
```
...is the correct syntax for unsetting an attribute.

Reply:
I've just checked in the AsciiDoc reference and both is actually allowed. Please, ignore this :)